libs/api-interfaces/src/lib/aggregate-views/view-models/resources/transcribed-audio.view-model.interface.ts

```diff
@@ -0,0 +1,18 @@
+import { IBaseViewModel } from '../base.view-model.interface';
+
+export interface ITranscribedAudioViewModel extends IBaseViewModel {
+    audioURL: string;
+
+    start: number;
+
+    length: number;
+
+    /**
+     * We may eventually want to make this a union of different
```

Comment:
Yes, sounds good

Reply:
I've added [this story](https://www.pivotaltracker.com/story/show/183728579); Let's tag the comment.